### PR TITLE
Erg grid columns

### PR DESCRIPTION
**Pull request by derpdead**

### CODE REVIEW

File: modules/@ergo/products/config/extends.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+export default {
+    dictionaries: [
+        {
+            stateProp: 'privileges',
+            dataFormat: {},
+            requestPath: '/dictionary/privileges',
+        },
+    ],
+};
```
Comment (bleto) [inline modules/@ergo/products/config/extends.js:14]: why it is here? Privileges dictionary is used in the User module.

File: report.20200406.073210.658.0.001.json [hunk 2]
```diff
@@ -0,0 +1,787 @@
+
```
Comment (bleto) [inline report.20200406.073210.658.0.001.json:1]: what is this file? I think it shouldn't be inserted to repository.

File: modules/@ergo/users/components/Grid/Layout/Table/Cells/Data/GridTextHintDataCell.vue [hunk 3]
```diff
@@ -0,0 +1,55 @@
+/*
```
Comment (bleto) [inline modules/@ergo/users/components/Grid/Layout/Table/Cells/Data/GridTextHintDataCell.vue:1]: Is this file is in the correct place? All components related with cell data are included in @Core module.

File: modules/@ergo/users/components/Grid/Layout/Table/Cells/Data/GridCheckDataCell.vue [hunk 4]
```diff
@@ -0,0 +1,57 @@
+/*
```
Comment (bleto) [inline modules/@ergo/users/components/Grid/Layout/Table/Cells/Data/GridCheckDataCell.vue:1]: Is this file is in the correct place? All components related with cell data are included in @core module.

File: modules/@ergo/users/components/Grid/Layout/Table/Cells/Data/GridTextHintDataCell.vue (hunk 3 above)
Comment (derpdead) [reply modules/@ergo/users/components/Grid/Layout/Table/Cells/Data/GridTextHintDataCell.vue:1]: It's related to @users module only. I'll remember to do sth about it

File: modules/@ergo/users/components/Grid/Layout/Table/Cells/Data/GridCheckDataCell.vue (hunk 4 above)
Comment (derpdead) [reply modules/@ergo/users/components/Grid/Layout/Table/Cells/Data/GridCheckDataCell.vue:1]: It's releated to @users only

**Pull request merged**